cli/src/commands/mod.rs

```diff
@@ -29,6 +29,7 @@ mod edit;
 mod evolog;
 mod file;
 mod fix;
+mod gerrit;
```

Comment:
Do we want to guard Gerrit support behind a `gerrit` Cargo feature?

Reply:
No, I don't think it's worth it. If there was a big cost to carrying it for developers (e.g. maybe needing a hundred dependencies or some really complex 3rd party dependency thing) then that would be one thing, but in this case, I think it has a pretty minimal overall footprint, and reuses stuff we already have. I don't think a GitHub or GitLab backend would be worth splitting out either fwiw; in practice everyone would just enable it anyway, probably.

(More broadly, features just add bloat to the build matrix and tend to create edge cases. IMO we should try to avoid them if possible, not just in this case.)

Reply:
That makes sense. I agree.

I think it might still be worth it for GitHub/GitLab if the build dependencies for them turned out to be large, but we can discuss that later when someone sends a PR to add support for them.